core/src/main/java/org/apache/carbondata/core/util/path/CarbonTablePath.java

```diff
@@ -409,11 +433,10 @@ public static String getTimeStampFromFileName(String carbonDataFileName) {
      * @return
      */
     public static Boolean compareCarbonFileTimeStamp(String fileName, Long timestamp) {
-      int lastIndexOfHyphen = fileName.lastIndexOf("-");
-      int lastIndexOfDot = fileName.lastIndexOf(".");
-      if (lastIndexOfHyphen > 0 && lastIndexOfDot > 0) {
-        return fileName.substring(fileName.lastIndexOf("-") + 1, fileName.lastIndexOf("."))
-            .equals(timestamp.toString());
+      int startIndex = fileName.lastIndexOf("-");
```

Comment:
Suggest to use CarbonCommonConstants for "-". "."

Reply:
This func is not used anywhere, I deleted it